test/e2e/admission_controller/admission_controller_suite_test.go

```diff
@@ -188,6 +190,8 @@ func TestAdmissionController(t *testing.T) {
 }
 
 var _ = BeforeSuite(func() {
+	_, filename, _, _ := runtime.Caller(0)
+	suiteName = common.GetSuiteName(filename)
```

Comment:
These two lines are constantly repeated. It would make sense to rename and extend `GetSuiteName()` to return two strings, the `suiteName` and `filename`, eg. `common.GetTestSuiteInfo()`.